README.md

```diff
@@ -2,6 +2,72 @@
 
 `image-tools` is a collection of tools for working with the [OCI image format specification](https://github.com/opencontainers/image-spec).
 
+# Install
+
+Recommend to use `go` to download.
+
+```
+$ go get github.com/opencontainers/image-tools/
+$ cd $GOPATH/src/github.com/opencontainers/image-tools/
+$ make all
+$ sudo make install
+```
+
+## uninstall
+
+```
+$ sudo make uninstall
+```
+
+## example
+
+Firstly, May be you need to install `skopeo`  and more information at [here](https://github.com/projectatomic/skopeo).
```

Comment:
All of these examples are using a `busybox-oci` image.  I'd probably structure it like:

### Obtaining an image

The following examples assume you have a [image-layout][image-layout] tar archive at `busybox-oci`.
One way to acquire that image is with [skopeo][]:

```
$ skopeo copy docker://busybox oci:busybox-oci
```

### oci-create-runtime-bundle

```
$ mkdir busybox-bundle
$ oci-create-runtime-bundle --ref latest busybox-oci busybox-bundle
$ cd busybox-bundle && sudo runc run busybox
```

More information about `oci-create-runtime-bundle` can be found in its [man page](./cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md).

…

[image-layout]: https://github.com/opencontainers/image-spec/blob/v1.0.0-rc2/image-layout.md
[skopeo]: https://github.com/projectatomic/skopeo#installing